### ISSUE TITLE
This doesn't work

### ISSUE THREAD

**Issue by krzysztofMajchrzak-GIT** [JiaoJingPing/siamese_cnn#1]
After full training it is only overfitted. The loss is aroud 0.1 but the accuracy is around 50% so it's just a coin flip it does not work at all.

**Comment by AGenchev** [comment]
Interesting. I didn't test it, but I am interested in building something similar with ResNetXX backbone. Which loss function did you use ?
